test/unit/reductions/grid_search/test_grid_search_arguments.py

```diff
@@ -231,6 +232,40 @@ def test_grid_size_warning_up_to_5_sensitive_feature_group(self, transformX, tra
         else:
             assert len(log_records) == 0
 
+    @pytest.mark.parametrize("transformX", candidate_X_transforms)
+    @pytest.mark.parametrize("transformY", candidate_Y_transforms)
+    @pytest.mark.parametrize("transformA", candidate_A_transforms)
+    def test_custom_grid(self, transformX, transformY, transformA):
+
+        grid_size = 10
+        grid_limit = 2.0
+        grid_offset = 0.1
```

Comment:
I'm guessing that the `grid_offset` is the key? If so, can you add a comment to that effect

Reply:
Nope, these values don't matter, I'm just using the defaults.